### DISCUSSION TITLE
Torus Wallet Connector Plugin chat hide

### DISCUSSION THREAD

**Discussion by infocodiste** [Web3Auth/Web3Auth#692] [Help]
I am using Torus Wallet Connector Plugin
on clicking the torus wallet icon, it also opens the chat icon at the bottom right corner. I want to hide that chat icon. How to do it?

**Comment by KetulCodiste** [comment]
Can anyone please answer this question quickly. It is an urgent requirement

**Comment by rinzler99** [comment]
Hey, sorry for delay in answering your query. I had few questions:
1. Which chain are you working?
2. If possible, Can you share you code snippet or the screenshot of the same.

**Reply by infocodiste** [reply]
I am working on polygon.

const torusPlugin = new TorusWalletConnectorPlugin({
        torusWalletOpts: {
          buttonPosition: "top-right"
        },
        walletInitOptions: {
          whiteLabel: {
            theme: { isDark: true, colors: { primary: "#00a8ff" } },
            logoDark: "https://web3auth.io/images/w3a-L-Favicon-1.svg",
            logoLight: "https://web3auth.io/images/w3a-D-Favicon-1.svg",
          },
          useWalletConnect: true,
          enableLogging: true,
          showTorusButton: false
        },
      });
      await web3Auth.addPlugin(torusPlugin);

![Screenshot 2022-10-12 at 10 52 29 AM](https://user-images.githubusercontent.com/78797181/195257047-71c5356a-ee50-4f2c-b7de-e7b48aa15f40.png)

You can see that there is a chat icon at the bottom right corner and I want to remove/hide it. How to do it?

**Reply by rinzler99** [reply]
Hey, Thanks for your patience. Team is working on to add an option to hide chat support icon from torus embed. Sometimes, this chat icon popups due to some error or region based. Please can you share console logs here? Thanks in advance!

**Reply by infocodiste** [reply]
Now I am not getting this chat icon. Did you guys fix this issue in the new patch version?

**Comment by rinzler99** [comment]
We haven't merge the fix yet. Sometimes it does come because of region based issues. Once it's merged devs can use param to show/hide it in the app. Thanks for your patience!

**Reply by KetulCodiste** [reply]
Can you please tell me that approximately how much days it will take to merge this changes?

**Comment by reecejohnson** [comment]
I'm having the same issue. 

The chat icon is persistent and I can't find any way to hide it.

Any update of the progress here? 

Cheers

**Comment by ihsraham** [comment]
@reecejohnson @infocodiste Please let me know if you still face this issue since I am not able to find the persistent chat icon in the examples.
